docs/docs/reference/contextual/extension-methods.md

```diff
@@ -89,17 +89,31 @@ List("a", "bb", "ccc").sumBy[Int](_.length)
 ```
 
 By contrast, type arguments matching type parameters following `extension` can be passed
-only if the method is referenced as a regular method:
-
-```scala
-sumBy[String](List("a", "bb", "ccc"))(_.length)
-```
-
-or, passing, both type arguments
-
-```scala
-sumBy[String](List("a", "bb", "ccc"))[Int](_.length)
-```
+in two different ways:
+
+- When the method is referenced as an extension method:
+
+  ```scala
+  List("a", "bb", "ccc").sumBy[Int](_.length)
+  ```
+
+  Or, when passing both type arguments:
+  
+  ```scala
+  List[String]("a", "bb", "ccc").sumBy[Int](_.length)
```

Comment:
I believe that this is actually the same as the form directly above, i.e. `[String]` is an argument to `List.apply` not `sumBy`, so really you are still only passing one type parameter list to `sumBy`.

Reply:
Yes, you are right. I just update this PR and remove that inexact description.